client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx

```diff
@@ -39,6 +48,7 @@ const actionLinks = [
 ];
 
 export const ChaptersPage: NextPageWithLayout = () => {
+  const [filterChapter, setFilterChapter] = useState('');
```

Comment:
did you know that the type will fail if it's `{filterChapter, setFilterChapter}`. I have spent a good while on it, trying to use generics, until I gave up and copied `hasMore` into it

Reply:
`hasMore`? The code you've written seems fine as is.